build.sbt

```diff
@@ -115,7 +114,7 @@ lazy val munit              = "org.scalameta"                %% "munit"
 lazy val nimbusJoseJwt      = "com.nimbusds"                  % "nimbus-jose-jwt"          % nimbusJoseJwtVersion
 lazy val pureconfig         = "com.github.pureconfig"        %% "pureconfig"               % pureconfigVersion
 lazy val pureconfigCats     = "com.github.pureconfig"        %% "pureconfig-cats"          % pureconfigVersion
-lazy val scalaLogging       = "com.typesafe.scala-logging"   %% "scala-logging"            % scalaLoggingVersion
+lazy val scalaReflect       = "org.scala-lang"                % "scala-reflect"            % scalaCompilerVersion
```

Comment:
I had to introduce this one as it was a transitive dependency from scalalogging that is needed for Magnolia